Styling/README.md

```diff
@@ -1309,6 +1309,20 @@ For example:
 
 **TODO : add note about GLSL implementations requires strict type comparisons among other things: https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/140**
 
+#### Vector Data
+
+A [Vector Tile](../TileFormats/VectorData/README.md) is a collection of points, polygons, and polylines. All vector data can be styled similar to other features with `color` and `show`. Points have several other styling options:
+
+| Style | Description | Default |
+| --- | --- | --- |
+| pointSize | The size of the point in pixels. | 8.0 |
+| image | A URL to an image to be displayed instead of a point. | " " |
```

Comment:
I could be a data URI. 

Would another option for the future be to have an atlas as part of the payload and the image be a subregion of the atlas? That way implementations wouldn't have to rebuild an atlas.